scripting/include/get5.inc

```diff
@@ -81,7 +81,7 @@ native bool Get5_LoadMatchConfig(const char[] filename);
 // Loads a match config by downloading a remote config via a HTTP GET request.
 // Request parameters should be left out of the url and put into the optional ArrayLists parameters.
 native bool Get5_LoadMatchConfigFromURL(const char[] url, ArrayList paramNames = null,
-                                        ArrayList paramValues = null);
+                                        ArrayList paramValues = null, const char[] token = "");
```

Comment:
The parameter provided here is never read by `Native_LoadMatchConfigFromURL`. Please see how the arraylists are read using `GetNativeCell(x)` - the same should be applied to the header; and preferably a header key-value pair as two arguments, as I commented originally. The current implementation leaves this argument always unused.

Reply:
I should not do stuff without enough sleep.
I should not do stuff without enough sleep.
I should not do stuff without enough sleep.
:D